src/select.html

```diff
@@ -13,4 +13,7 @@
 	<li ng-mousedown="updateModel(item)" ng-repeat="item in ::collection" ng-class="{'+selected': (selectedIndex === $index)}">
 		<a>{{appendText}} {{isString(item.value) ? item.value : item}}</a>
 	</li>
+	<li ng-if="linkText && linkHref">
+		<a href="{{linkHref}}" class="cps-blue-link">{{linkText}}</a>
+	</li>
```

Comment:
Nope, this actually works fine. It's because it's calculated from the top, and this link is added to the bottom, so it doesn't affect the calculation.